### ISSUE TITLE
unpack! in gem_package.rb forgets to quote paths.

### ISSUE THREAD

**Issue by hlian** [defunkt/rip#52]
The current code is:

```
system "#{gembin} unpack #{cache_file} --target=#{packages_path} > /dev/null"
```

This fails when `cache_file` or `packages_path` or `gembin` contain spaces. A simple fix would be

```
system "\"#{gembin}\" unpack \"#{cache_file}\" --target=\"#{packages_path}\" > /dev/null"
```

**Comment by cldwalker** [comment]
Here's a patch for this: cldwalker@1afdc0fbd814e42e30f35014fa50d5f6f7fc7e81

**Comment by defunkt** [comment]
Word. Merged in as d22be3af1d55f8e8c519004799f39ceba8115671

**Comment by hlian** [comment]
Hooray. Thanks everybody.
